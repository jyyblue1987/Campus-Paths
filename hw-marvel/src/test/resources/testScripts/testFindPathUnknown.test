# Test find a path on an unknown node

LoadGraph name nameYear.csv

FindPath name GISELLE-VANESSA TOM-CRUISE