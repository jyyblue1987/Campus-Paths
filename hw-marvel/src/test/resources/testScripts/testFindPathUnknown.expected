# Test find a path on an unknown node

loaded graph name

unknown: TOM-CRUISE